.github/workflows/build.yml

```diff
@@ -76,7 +76,7 @@ jobs:
         run: |
           # Generate a timestamp for creating an image version history
           TIMESTAMP="$(date +%Y%m%d)"
-          MAJOR_VERSION="$(echo ${{ steps.labels.outputs.VERSION }} | cut -d . -f 1)"
+          MAJOR_VERSION="${{ env.FEDORA_MAJOR_VERSION }}"
```

Comment:
This does not seem to be the correct thing to fix.  `env.FEDORA_MAJOR_VERSION` is the version tag in the recipe, and disabling it makes it so that a setting of `latest` there and there are no version tags, which breaks netinstall ISOs.
What I am gathering here is that sometimes (always?) the version gathered on line 70 is empty, and as such causes line 87 to add the following commit tags: `"<empty>" "<empty>-${TIMESTAMP}` (with empty representing the empty major version variable.
What should actually be done is somehow find the root cause of this issue, which I'll start investigating right away.

If someone desperately stumbles upon this PR and does not care about ISOs, you can implement this fix for yourself for the time being.

Reply:
We can alleviate this by only adding the major version tags if the major version variable is not empty, which I'll look into implementing with another PR.

Reply:
Looking at the Bluefin image's labels with skopeo reveals this to be the true cause.
```json
{
  "io.artifacthub.package.logo-url": "https://avatars.githubusercontent.com/u/120078124?s=200&v=4",
  "io.artifacthub.package.readme-url": "https://raw.githubusercontent.com/ublue-os/bluefin/bluefin/README.md",
  "io.buildah.version": "1.23.1",
  "org.opencontainers.image.created": "2023-08-15T10:02:40.119Z",
  "org.opencontainers.image.description": "An interpretation of the Ubuntu spirit built on Fedora technology",
  "org.opencontainers.image.licenses": "Apache-2.0",
  "org.opencontainers.image.revision": "f7652c29d50328c3754bdee2123c01531e9ca6f5",
  "org.opencontainers.image.source": "https://github.com/ublue-os/bluefin",
  "org.opencontainers.image.title": "bluefin",
  "org.opencontainers.image.url": "https://github.com/ublue-os/bluefin",
  "org.opencontainers.image.version": "", # standard version string parsed by startingpoint, empty
  "ostree.bootable": "true",
  "ostree.commit": "e333bf9858fa928170a97cb349117c812039d5b65920a35e56b2b36c2e7df522",
  "ostree.final-diffid": "sha256:12787d84fa137cd5649a9005efe98ec9d05ea46245fdc50aecb7dd007f2035b1",
  "ostree.linux": "6.4.10-200.fc38.x86_64",
  "quay.expires-after": "4w",
  "rpmostree.inputhash": "12795892b5e7a226b9576c14f6c1974a8f26f8035054efd864b8eb31900b871a",
  "version": "38.20230815.0", # bluefins version string
}
```